src/index.html

```diff
@@ -1,12 +1,287 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="page__header header">
+      <a href="#" class="header__logo">
+        <img src="moyo.jpg" alt="Logo"
+         class="header__logo-image">
```

Comment:
Fix code style everywhere
```suggestion
        <img
          src="moyo.jpg"
          alt="Logo"
          class="header__logo-image"
        >
```